src/Components/ToggleableContactForm.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Created by brianmichael on 2/3/17.
+ */
+import React, {Component} from 'react';
+import ContactForm from './ContactForm';
+
+class ToggleableContactForm extends Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleFormOpen = this.handleFormOpen.bind(this);
+    this.handleFormClose = this.handleFormClose.bind(this);
+
+  }
+
+  handleFormOpen() {
+    this.setState({
+      isOpen: true
+    });
+  }
+
+  handleFormClose() {
+    this.setState({
+      isOpen: false
+    });
+  }
+
+
+  render() {
+
+    if (this.state.isOpen) {
+      return (
+        <ContactForm
+          contact={this.props.contact}
+          handleAddContactSubmit={this.props.handleAddContactSubmit}
+          onInputChange={this.props.onInputChange}
+          validationErrors={this.props.validationErrors}
+          handleFormClose={this.handleFormClose}
+           />
```

Comment:
Closing of the self closing tag over multiple lines should happen on the same indentation level where it starts.